packages/devtools_testing/lib/support/flutter_test_environment.dart

```diff
@@ -48,25 +47,25 @@ class FlutterTestEnvironment {
 
   // This function will be called after we have ran the Flutter app and the
   // vmService is opened.
-  VoidAsyncFunction _afterNewSetup;
```

Comment:
if you manage to get package:flutter to stop using VoidAsyncFunction and VoidFunction then I'll approve it here. :)

Reply:
There are many best practices the Flutter SDK decided not to follow...

Reply:
`package:flutter` does not use `VoidAsyncFunction`. It does use `AsyncCallback` in exactly 1 place, and it makes a typedef with that name public.

It also does not use `VoidFunction`. It does use `VoidCallback`.